nisystemlink/clients/auth/models/_auth_models.py

```diff
@@ -0,0 +1,373 @@
+    )
+    """
+    The id of the policy template. Only set if the policy has been created based on a template and
+    does not contain inline statements.
+    """
+    workspace: Optional[str] = Field(None, example="workspace-id")
+    """
+    The workspace the policy template applies to. Only set if the policy has been created based on a
+    template and does not contain inline statements.
+    """
+
+
+class Key(JsonModel):
+    id: Optional[str] = Field(None, example="api-key-id")
+    """
+    The unique id
+    """
+    name: Optional[str] = Field(None, example="api-key-name")
+    """
+    The key's name
+    """
+    user_id: Optional[str] = Field(None, alias="userId", example="user-id")
+    """
+    The user id
+    """
+    created: Optional[datetime] = Field(None, example="2019-12-02T15:31:45.379Z")
+    """
+    The created timestamp
+    """
+    updated: Optional[datetime] = Field(None, example="2019-12-02T15:31:45.379Z")
+    """
+    The last updated timestamp
+    """
+    expiry: Optional[datetime] = Field(None, example="2019-12-02T15:31:45.379Z")
+    """
+    The time when the key expires (epoch in milliseconds)
+    """
+    enabled: Optional[bool] = Field(None, example=True)
+    """
+    Whether the key is enabled or not
+    """
+    deleted: Optional[bool] = Field(None, example=True)
+    """
+    Whether the key is deleted or not
+    """
+    default_workspace: Optional[str] = Field(
+        None, alias="defaultWorkspace", example="workspace-id"
+    )
+    """
+    This field overrides the default workspace when authenticating.
+    """
+    properties: Optional[Dict[str, str]] = Field(None, example={"key1": "value1"})
+    """
+    A map of key value properties
+    """
+    policies: Optional[List[Policy]] = None
+    """
+    A list of policy definitions including statements and permissions
+    """
+
+
+class Status(Enum):
+    PENDING = "pending"
+    ACTIVE = "active"
+
+
+class User(JsonModel):
+    id: Optional[str] = Field(None, example="user-id")
+    """
+    The unique id
+    """
+    first_name: Optional[str] = Field(
+        None, alias="firstName", example="user-first-name"
+    )
+    """
+    The user's first name
+    """
+    last_name: Optional[str] = Field(None, alias="lastName", example="user-last-name")
+    """
+    The user's last name
+    """
+    email: Optional[str] = Field(None, example="example@email.com")
+    """
+    The user's email
+    """
+    phone: Optional[str] = Field(None, example="555-555-5555")
+    """
+    The user's contact phone number
+    """
+    niua_id: Optional[str] = Field(None, alias="niuaId", example="example@email.com")
+    """
+    The external id (niuaId, SID, login name)
+    """
+    login: Optional[str] = None
+    """
+    The login name of the user. This the "username" or equivalent entered when
+    the user authenticates with the identity provider.
+    """
+    accepted_to_s: Optional[bool] = Field(None, alias="acceptedToS", example=True)
```

Comment:
I know we're not using this model for anything yet, but `accepted_to_s` is a weird name. Probably should just be `accepted_tos`